code/modules/research/xenobiology/xenobiology.dm

```diff
@@ -848,10 +848,10 @@
 		var/obj/vehicle/V = C
 		var/datum/component/riding/R = V.GetComponent(/datum/component/riding)
 		if(R)
-			if(R.vehicle_move_delay <= 0 )
+			if(R.vehicle_move_delay <= (CONFIG_GET(number/movedelay/run_delay) * 1.15) )
```

Comment:
```suggestion
			var/vehicle_speed_mod = (CONFIG_GET(number/movedelay/run_delay) * 1.15)
			if(R.vehicle_move_delay <= vehicle_speed_mod )
```
Doing this means it only pulls from the config once instead of twice.

Reply:
Also doesn't bigger number = slower? Not a very good speed potion if it slows your vehicle lol.